pkg/apis/core/validation/validation.go

```diff
@@ -4775,6 +4798,15 @@ func ValidatePodUpdate(newPod, oldPod *core.Pod, opts PodValidationOptions) fiel
 // ValidateContainerStateTransition test to if any illegal container state transitions are being attempted
 func ValidateContainerStateTransition(newStatuses, oldStatuses []core.ContainerStatus, fldpath *field.Path, restartPolicy core.RestartPolicy) field.ErrorList {
 	allErrs := field.ErrorList{}
+
+	if !utilfeature.DefaultFeatureGate.Enabled(features.SupplementalGroupsPolicy) {
```

Comment:
same as above, except this is not required

Reply:
Actually, I am not sure there's any validation you CAN do here.  Maybe check thart UID and GIDs are not negative?